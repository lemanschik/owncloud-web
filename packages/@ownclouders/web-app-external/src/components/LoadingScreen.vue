<template>
  <div class="oc-text-center">
    <oc-spinner size="xlarge" />
    <p v-translate class="oc-invisible">Loading app</p>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue'
export default defineComponent({
  name: 'LoadingScreen'
})
</script>
