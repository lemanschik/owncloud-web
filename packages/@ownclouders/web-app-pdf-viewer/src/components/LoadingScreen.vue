<template>
  <div class="oc-position-center oc-text-center">
    <oc-spinner size="xlarge" />
    <p v-translate class="oc-invisible">Loading PDF document</p>
  </div>
</template>

<script lang="ts">
import { defineComponent } from 'vue'

export default defineComponent({
  name: 'LoadingScreen'
})
</script>
