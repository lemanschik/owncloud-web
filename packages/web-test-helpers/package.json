{
  "name": "web-test-helpers",
  "private": true,
  "main": "src/index.ts",
  "peerDependencies": {
    "@casl/ability": "^6.3.3",
    "@casl/vue": "^2.2.1",
    "axios": "0.27.2",
    "design-system": "@ownclouders/design-system",
    "vue3-gettext": "^2.3.3",
    "vue-router": "4.1.6",
    "vuex": "4.1.0"
  }
}
