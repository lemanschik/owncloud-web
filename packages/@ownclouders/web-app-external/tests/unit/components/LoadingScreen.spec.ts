import LoadingScreen from '../../../src/components/LoadingScreen.vue'
import { defaultPlugins, mount } from 'web-test-helpers'

describe('The external app loading screen component', () => {
  test('displays a spinner and a paragraph', () => {
    const wrapper = mount(LoadingScreen, {
      global: {
        stubs: {
          OcSpinner: true
        },
        plugins: [...defaultPlugins()]
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
