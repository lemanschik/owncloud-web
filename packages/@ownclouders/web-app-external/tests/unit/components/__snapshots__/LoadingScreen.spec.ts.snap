// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The external app loading screen component displays a spinner and a paragraph 1`] = `
<div class="oc-text-center">
  <oc-spinner-stub arialabel="" size="xlarge"></oc-spinner-stub>
  <p class="oc-invisible" data-current-language="en" data-msgid="Loading app">Loading app</p>
</div>
`;
